configure-linux.sh

```diff
@@ -300,6 +308,7 @@ if (( $COMPILE_WITH_DPDK > 0 )) ; then
 
    # if DPDK ver < 20.11 concat additional definitions to PcapPlusPlus.mk
    CUR_DPDK_VERSION=$(get_dpdk_version)
+   echo $CUR_DPDK_VERSION
```

Comment:
Is this a leftover from your debugging sessions? I'm not sure we need this print 🤔

Reply:
Yes, sorry.